pkg/controller/migplan/migplan_controller.go

```diff
@@ -379,6 +387,48 @@ func (r *ReconcileMigPlan) planSuspended(plan *migapi.MigPlan) error {
 	return nil
 }
 
+func (r *ReconcileMigPlan) AddRunningLabelForPlan(plan *migapi.MigPlan) error {
+	migrations, err := plan.ListMigrations(r)
+	if err != nil {
+		return liberr.Wrap(err)
+	}
+
+	if len(migrations) == 0 {
+		if plan.Labels == nil {
+			plan.Labels = make(map[string]string)
+		}
+		plan.Labels["migplan-running"] = "false"
+		err = r.Update(context.TODO(), plan)
+		if err != nil {
+			return liberr.Wrap(err)
+		}
```

Comment:
`return nil`
here and remove the else block underneath

Reply:
Done